kvtests/common_tests.go

```diff
@@ -302,6 +302,42 @@ func TestRange(t *testing.T, storeProvider StoreProvider) {
 	})
 }
 
+func TestEmpty(t *testing.T, storeProvider StoreProvider) {
```

Comment:
maybe include a test that has data in a different shelf to make sure it only tests the correct shelf?